### PR TITLE
Allow watching js tests with novnc

### PR DESCRIPTION
**Pull request by bboro**
#586 [2h]

### CODE REVIEW

**Comment by bboro** [comment]
https://github.com/Gizra/drupal-starter/assets/24897663/6893b7c5-b58b-4f29-ab3c-d36737415ad7

TB: 2.3 / 2h

File: README.md [hunk 1]
```diff
@@ -190,6 +190,10 @@ example implementation. By extending the above base class you can also take scre
 **Note: You should not leave calls to `takeScreenshot` in the codebase when committing, this is meant only for
 local debugging purposes.**
 
+You can also watch what the tests are doing in the browser using noVNC. To do so, simply open a browser and open
+https://drupal-starter.ddev.site:7900 and click Connect. The password is `secret`. Now simply run the tests
+and you can see the test running in the browser.
+
```
Comment (bboro) [inline README.md:196]: @mgurjanov @dipakmdhrm FYI ^

**Review by AronNovak — APPROVED** [review]
Looks good! And a great debugging path, for sure!

**Comment by dipakmdhrm** [comment]
> Looks good! And a great debugging path, for sure!

+1. Just used this to set up a JS test [in portal](https://github.com/Gizra/jep-portal/pull/2661)

**Pull request merged**